projects/2019_12_16_MultiSig_QT_Wallet.md

```diff
@@ -0,0 +1,90 @@
+# Multi-Sig QT Wallet Project
+* Project Name: MultiSig_QT_Wallet
+* Contact Email: vishnu@gridfinity.com
+* Project participants:
+  * Vishnu Seesahai - Project Owner
+  * *Other participants will be added as needed*
+* Projected duration: 3 months
+* Projected effort: 1 person per month = 3 person-months
+* Requested PKT contribution: 40 million PKT
+* Project status: **PROPOSED**
+
+## Project summary
+
+The objectives of the Multi-Sig QT Wallet is to expand the PKT network adoption by users who don’t have the resources for using the current [text wallet](https://github.com/pkt-cash/pktwallet), increase the network's density of nodes, and improve security & control by the implementation of multi-signature.
+
+## 1. Project Relevance to the PKT Network
+> _"The [objective of The Network Steward](https://pkt-cash.github.io/www.pkt.cash/steward/) is to reduce barriers of entry to network / VPN providers in the pursuit of increased individual autonomy and privacy"_
+
+The creation of a new multi-sig QT PKT wallet can achieve the following benefits for the PKT Network:
+
+* Expand the adoption of the PKT Network to users who do not have the technical skills, time, or resources to operate the current [open-source files of a wallet](http://github.com/pkt-cash/pktwallet), and run a [full node](https://github.com/pkt-cash/pktd).
+* Higher security & control, backup feature, and joint-account capability, provided by the multi-sig-transaction accessibility.
+* Increase the density of the PKT Network nodes, to contribute to building the infrastructure of a functional decentralized network.
+
+In the list of examples of [Projects Supported by The Network Steward](http://github.com/pkt-cash/ns-projects/blob/master/readme.md), the first example is a wallet. As of the date of preparation of this proposal, the only available wallet is a [text-based wallet](http://github.com/pkt-cash/pktwallet).
+
+## 2. Project Deliverables
+The project will create new software for a standalone application, which should be downloaded and installed from an Installer. The installation will come bundled and ready to run a full-node, [PKTD](https://github.com/pkt-cash/pktd), as well as a [PKT Wallet](http://github.com/pkt-cash/pktwallet) instance.
+
+This deliverables are:
+* New software for a standalone application.
+* The software will be available under the [ISC](https://opensource.org/licenses/ISC) license.
+* Any artistic non-software assets will be available under the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
+* The maintainer of this software will be: Vishnu Seesahai
+* The software will be hosted on https://github.com/pkt-cash, if the network steward wishes it so.
+
+### 2.1 Development overview
+The development will be carried out by using the QT Software toolkit due to its history of reliability, portability, and low memory usage.
+
+The wallet will have a graphical user interface (GUI).
+Compatible operating systems: MacOS, Unix.
```

Comment:
@cjdelisle I wonder what platform are people expected to run full nodes. Probably not MacOS based systems right?

If this is written in QT can it be targeted for different operating systems, with packages for Debian / Ubuntu, which I expect a lot of full nodes to use?

Reply:
At this stage, we're not to the point where the chain is so huge that it's unreasonable to run a full node on one's laptop. Also because @artrepreneur is planning to use pktwallet codebase, whatever he develops will be forward compatible with the neutrino-backed light version of pktwallet whenever that gets up and running.

Reply:
@cjdelisle probably MAcOS will be the least likely full-node system, but QT does compile for MacOS